ROADMAP.md

```diff
@@ -0,0 +1,78 @@
+# k6 open-source project's roadmap
+
+Our team is dedicated to continuously improving k6 and providing the best user experience possible. We're excited to share our roadmap with the k6 community, outlining the upcoming features and improvements we have planned.
+
+This roadmap covers user-oriented features, UX improvements, JavaScript support, and k6 internals that our team will focus on. Remember that timeframes and priorities may shift, but we believe it's important to share our vision and allow users to plan accordingly.
+
+We hope this updated roadmap provides a clear overview of our plans for k6's future development. As always, we welcome feedback, corrections, and suggestions to make this roadmap more comprehensive, accessible, and valuable for the k6 community.
+
+## Short-term goals
+
+These are goals that are tentatively achievable by **Q2 2023**.
```

Comment:
```suggestion
These are goals that are tentatively achievable by **Q2 2023** (April-June).
```

I think it is better for us to be explicit in at least one place what quarter is as unfortunately it is shifted in some cases.